flaskApi/parser.py

```diff
@@ -0,0 +1,64 @@
+from pptx import Presentation
+
+
+def get_presentation_as_list_of_slides(path_to_presentation):
+    """
+    Gets   pptx presentation and read it and return it as list of slides
+
+   Parameters
+   ----------
+   path_to_presentation : str
+        path to pptx presentation
+
+   Returns
+   -------
+   list
+        return the presentation as list of dicts
+        example:
+        [
+           mocks\n
+           Python mocks are objects that ' ,
+
+         ' Refactoring code\n
+           The process of improving the design ...' ,
+           ...
+       ]
+   """
+    presentation = Presentation(path_to_presentation)
+
+    presentation_as_list_of_slides = [get_text_from_slide(slide) for slide in presentation.slides]
+    return presentation_as_list_of_slides
+
+
+def get_text_from_slide(slide):
```

Comment:
Separate code to small functions.  Read more: https://softwareengineering.stackexchange.com/questions/308108/when-is-it-appropriate-to-make-a-separate-function-when-there-will-only-ever-be